tests/unit/test_cli.py

```diff
@@ -332,6 +334,27 @@ def test_migrate_credentials_azure(ws):
         ws.storage_credentials.list.assert_called()
 
 
+def test_migrate_credentials_aws(ws, mocker):
+    mocker.patch("shutil.which", return_value=True)
+    ws.config.is_azure = False
+    ws.config.is_aws = True
+    uc_trust_policy = mocker.patch(
+        "databricks.labs.ucx.assessment.aws.AWSResourcePermissions.update_uc_role_trust_policy"
+    )
+    with patch("databricks.labs.blueprint.tui.Prompts.confirm", return_value=True):
+        migrate_credentials(ws, aws_profile="profile")
+        ws.storage_credentials.list.assert_called()
+        uc_trust_policy.assert_called_once()
+
+
+def test_migrate_credentials_aws_no_profile(ws, caplog, mocker):
+    mocker.patch("shutil.which", return_value="/path/aws")
+    ws.config.is_azure = False
+    ws.config.is_aws = True
+    migrate_credentials(ws)
+    assert any({"AWS Profile is not specified." in message for message in caplog.messages})
```

Comment:
this will give cryptic error message when failed. do exact string matches - `assert "..." in caplog.messages`

Reply:
fixed